CRM/Core/Payment.php

```diff
@@ -831,7 +831,7 @@ public function getPaymentFormFieldsMetadata() {
           'size' => 20,
           'maxlength' => 20,
           'autocomplete' => 'off',
-          'class' => 'creditcard',
+          'class' => 'creditcard, required',
```

Comment:
@seamuslee001 I don't think this should have a comma here?

Reply:
yep true pushing a commit now